guides/common/modules/con_provisioning-management-with-projectname.adoc

```diff
@@ -0,0 +1,7 @@
+[id="Provisioning-Management-with-{ProjectNameID}_{context}"]
+= Provisioning Management with {ProjectName}
+
+With {ProjectName}, you can provision hosts on various platforms with many provisioning methods from a centralized console.
```

Comment:
I am unsure about the word "platforms" here. Maybe it's perfect for users new to Foreman? Or maybe we should stick to "compute resources" and/or "clouds"?